packages/blade/src/utils/useCallbackRef.ts

```diff
@@ -1,5 +1,8 @@
 /* eslint-disable @typescript-eslint/explicit-function-return-type */
 import { useCallback, useInsertionEffect, useRef } from 'react';
+import { useIsomorphicLayoutEffect } from './useIsomorphicLayoutEffect';
+
+const useInsertionEffectFallback = useInsertionEffect || useIsomorphicLayoutEffect;
```

Comment:
Wouldn't react 17 fail with useInsertionEffect not defined or no import error instead of it being undefined / falsy? 

Like does the function exist in react 17 but has falsy value?

Reply:
I'll check once, with React17, otherwise we can just do React.useInsertionEffect instead of importing it.

Reply:
Tested, worked in Vite failed in Webpack. 

Fixed it in https://github.com/razorpay/blade/pull/2450/commits/4ea8acf3b52bd86549c695b5d22f673cfee4a9ac